test/expo/scripts/build-ios.sh

```diff
@@ -5,10 +5,7 @@ set -e
 # Lets make sure the build folder was cleared out correctly
 rm -rf $BUILDKITE_BUILD_CHECKOUT_PATH/build/*
 cd test/expo/features/fixtures/test-app
-npm i turtle-cli@0.14 bunyan
-perl -0777 -i.original -pe "s/entitlements\\['aps-environment'\\] =[^;]+;//gs" node_modules/\@expo/xdl/build/detach/IosNSBundle.js
```

Comment:
Have we confirmed that we no longer need to remove the push notification entitlements by running a build locally and then using the produced artefact in a test run?

Reply:
Resolved through Slack conversation.